packages/palette/src/elements/CulturalBadge/CulturalBadge.tsx

```diff
@@ -0,0 +1,20 @@
+import { Box, Text } from "@artsy/palette"
+import * as React from "react"
+
+export interface CulturalBadgeProps {
+  badgeCategory: string
+}
+
+export const CulturalBadge: React.FC<CulturalBadgeProps> = ({
+  badgeCategory,
+}) => {
+  return (
+    <>
+      <Box backgroundColor={"black100"} width={100} mb={1}>
+        <Text variant="xs" color="white" textAlign={"center"}>
+          {badgeCategory}
+        </Text>
+      </Box>
+    </>
+  )
```

Comment:
A few things:
* Can get rid of the outer fragment, not needed
* This can be a single `Text` tag, no need to wrap in a `Box`, `Text` accepts everything `Box` does.
* Don't hardcode the width but rather use padding. (`px`)
* The color should be `white100`
* Accept `children` instead of `badgeCategory`
* Remove the bottom margin (in general never include outer margins on a component like this, especially in palette)
* You can get rid of the interpolation `{"black100"}` becomes just `"black100"`

Reply:
Also: let's name this `Badge`. One can imagine these being used for other things. The `Cultural` prefix is too specific.

Reply:
Updated.